src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -33,7 +33,7 @@ trait IterableLike[+A, +C[X] <: Iterable[X]]
   /** Create a collection of type `C[A]` from the elements of `coll`, which has
     *  the same element type as this collection. Overridden in StringOps and ArrayOps.
     */
-  protected[this] def fromIterableWithSameElemType(coll: Iterable[A]): C[A] = fromIterable(coll)
+  protected[this] def fromIterableWithSameElemType(coll: Iterable[A]): C[A]
```

Comment:
Why should it? I see no particular reason why we'd want the binary type constructors in `fromIterableWithSameElemType`. This compiles: https://github.com/szeiger/collection-strawman/commit/0de8480fbd61a58c2b6b603827dbfb0b35b2789b